volumina/sliceSelectorHud.py

```diff
@@ -490,11 +505,111 @@ def createQuadViewStatusBar(self,
 
         self.addSpacing(20)
 
-        self.timeLabel = QLabel("Time:")
+        self.timeSpinBox = DelayedSpinBox(750)
+
+        self.timeStartButton = QPushButton("Start")
+        self.addWidget(self.timeStartButton)
+        self.timeStartButton.clicked.connect(self._onTimeStartButtonClicked)
+
+        self.timeSlider = QSlider(Qt.Horizontal)
+        self.timeSlider.setMinimumWidth(10)
+        self.timeSlider.setMaximumWidth(200)
+        self.addWidget(self.timeSlider)
+        self.timeSlider.valueChanged.connect(self._onTimeSliderChanged)
+
+        self.timeEndButton = QPushButton("End")
+        self.timeEndButton.setFixedWidth(4*self.timeControlFontSize)
+        self.timeStartButton.setFixedWidth(4*self.timeControlFontSize)
+        self.addWidget(self.timeEndButton)
+        self.timeEndButton.clicked.connect(self._onTimeEndButtonClicked)
+
+        self.timeLabel = QLabel("       Time:")
         self.addWidget(self.timeLabel)
 
-        self.timeSpinBox = DelayedSpinBox(750)
+        timeControlFont = self.timeSpinBox.font()
+        if self.timeControlFontSize > timeControlFont.pointSize():
+            timeControlFont.setPixelSize(self.timeControlFontSize)
+            self.timeStartButton.setFont(timeControlFont)
+            self.timeEndButton.setFont(timeControlFont)
+            self.timeLabel.setFont(timeControlFont)
+            self.timeSpinBox.setFont(timeControlFont)
+
         self.addWidget(self.timeSpinBox)
+        self.timeSpinBox.delayedValueChanged.connect(self._onTimeSpinBoxChanged)
+
+    def _onTimeStartButtonClicked(self):
+        try: # cropping applet
+            self.timeSpinBox.setValue(
+                self.parent().parent().parent().parent().topLevelOperatorView.MinValueT.value) # opCropSelection
+        except:
+            try: # annotations applet
+                self.timeSpinBox.setValue(
+                    self.parent().parent().parent().parent().topLevelOperatorView.Crops.value[
+                        self.parent().parent().parent().parent()._drawer.cropListModel[
+                            self.parent().parent().parent().parent()._selectedRow ].name ]["time"][0])
```

Comment:
A hack done very much in a hurry the day before our annual sfb1129 appearance.
"TimeStart" button is essential for efficiently demoing the training 
(essentially embedded ManualTrackingApplet)

I expected my second solution to show up here, but it obviously got lost in a stash...
will update.